### PR TITLE
Kevin/fix redirect popup (EU) 

### PR DESCRIPTION
**Pull request by kevinw-binary**
# Description

Summary
Fix redirect popup width to 80% on mobile screen 

Changes:
styled component with media query

## Type of change

-   [x] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/components/custom/_eu-redirect-modal.js [hunk 1]
```diff
@@ -26,10 +26,13 @@ const ModalCard = styled.div`
     flex-direction: column;
     border-radius: 6px;
     box-shadow: 2px 2px 10px rgba(0, 0, 0, 0.3);
-    width: 100%;
     max-width: 44rem;
     background: var(--color-white);
     padding: 2.4rem;
+
+    @media (max-width: 380px) {
```
Comment (mustofa-binary) [inline src/components/custom/_eu-redirect-modal.js:33]: can use ${device.mobileL}

File: src/components/custom/_eu-redirect-modal.js [hunk 2]
```diff
@@ -79,6 +82,12 @@ const Cancel = styled.span`
         background-color: rgba(0, 0, 0, 0.08);
     }
 `
+
+const StyledText = styled(Text)`
+    @media (max-width: 380px) {
```
Comment (mustofa-binary) [inline src/components/custom/_eu-redirect-modal.js:87]: same here

File: src/components/custom/_eu-redirect-modal.js [hunk 3]
```diff
@@ -79,6 +82,12 @@ const Cancel = styled.span`
...
+        font-size: 7px;
```
Comment (mustofa-binary) [inline src/components/custom/_eu-redirect-modal.js:88]: i think the font size is too small, the minimum text we had is 10px

**Review by mustofa-binary — CHANGES_REQUESTED** [review]

File: src/components/custom/_eu-redirect-modal.js (hunk 3 above)
Comment (kevinw-binary) [reply src/components/custom/_eu-redirect-modal.js:88]: I guess i just remove the media query for this styled text

**Review by mustofa-binary — APPROVED** [review]

**Pull request merged**